test/integration/replicaset/replicaset_test.go

```diff
@@ -617,9 +606,10 @@ func TestPodOrphaningAndAdoptionWhenLabelsChange(t *testing.T) {
 
 	// Orphaning: RS should remove OwnerReference from a pod when the pod's labels change to not match its labels
 	// Although we can access the pod directly, we should still access it via selector to ensure the selector is working
+	podClient := c.Core().Pods(ns.Name)
 	podSelector := labels.Set{"foo": "bar"}.AsSelector()
```

Comment:
Fetching the pods using selector should be not more necessary as the pods can be obtained from `createRSsPods`.

Reply:
I am wrong - we should still fetch the pods using selector as we expect the RS to create the pods itself.